RiotSharp/RateLimitedRequester.cs

```diff
@@ -40,6 +43,50 @@ public static new RateLimitedRequester Instance
             return GetResponse(request);
         }
 
+        public string CreatePostRequest(string relativeUrl, Region region, string body, List<string> addedArguments = null,
```

Comment:
Maybe rename `CreateRequest` to `CreateGetRequest` since other verbs are supported now.
Also, maybe reorder the methods so that the async version is directly after the sync version, that would give us:

  1. `CreateGetRequest`
  2. `CreateGetRequestAsync`
  3. `CreatePostRequest`
  4. `CreatePostRequestAsync`
  5. `CreatePutRequest`
  6. `CreatePutRequestAsync`